### ISSUE TITLE
AEAudioController not included in Files?

### ISSUE THREAD

**Issue by acsarraf** [TheAmazingAudioEngine/TAAE2#56]
Hi, 

I added the TAAE2 files as suggested by your comment here: https://github.com/TheAmazingAudioEngine/TheAmazingAudioEngine2/issues/9, but I can't seem to find the AEAudioController files included in the project I cloned from github, even though it is included in the Sample project. 

I attempted to add the AEAudioController files to the TAEE2.xcodeproj file but I was still unable to import them. I then added the AEAudioController files to my own project and imported them, but after running, I basically got errors all across my AmazingAudioEngine.h file saying that none of the files it is trying to import could be found. 

Not really sure what to do at this point, 

Thanks, Alan

**Comment by markjeschke** [comment]
@theraad, Without seeing any code in your project, here's a check off list that may help you:

1. Open the `TheAmazingAudioEngine.xcworkspace` not the `TAAESample.xcodeproj` or `TheAmazingAudioEngine.xcodeproj`.

2. Add the `TAAESample-Bridging-Header.h` bridging header, which should only include: `#import "AEAudioController.h"`
<img width="796" alt="bridging-header" src="https://cloud.githubusercontent.com/assets/1577928/26556961/ac2de6a6-4452-11e7-8a70-4a3260fb20f3.png">

3. Select your project in Xcode's Project Navigator, then click the "Build Settings" tab. Search for "Objective-C Bridging Header" and add the `TAAESample/TAAESample-Bridging-Header.h` in its value.
<img width="1141" alt="add-bridging-header-in-build-settings" src="https://cloud.githubusercontent.com/assets/1577928/26556954/a5e84854-4452-11e7-8ac9-909ddd8a863b.png">

Please let me know if this helps. Without the bridging header, your project won't be able to import the `AEAudioController.h` and utilize TAAE2. Oh, and be sure that your bridging path is set correctly. 

Take care,
Mark

P.S. I downloaded TAAE2 repo again, and it included the `AEAudioController.h` and `AEAudioController.m` in the classes folder. I'm unsure why you're not seeing them within the project.

<img width="584" alt="aeaudiocontroller-included" src="https://cloud.githubusercontent.com/assets/1577928/26557093/be4eed98-4453-11e7-8134-486dd13a8fe0.png">

**Comment by acsarraf** [comment]
Hi @ markjeschke,
I couldn't seem to get it to work with the way you suggested, but I did get it working by creating a new copy of my project and including all the necessary files in my project folder. I then changed the imports so that it would work. 

My Project structure includes TAAE2 like so :

![screen shot 2017-05-30 at 12 33 40 pm](https://cloud.githubusercontent.com/assets/28601705/26577194/3f245f80-4534-11e7-919e-43f138dd6d4e.png)

Thanks a lot for your help, was much appreciated.

Alan
